src/vue_isotope.js

```diff
@@ -91,20 +91,22 @@
           return;
 
         this.listen()
-         
-        this.iso.remove(removed)
-        this.iso.insert(added)
-        this.iso._requestUpdate()      
+
```

Comment:
I found that checking for this.iso was necessary when using vue-router and changing routes. Without it I get the error `TypeError: Cannot read property 'remove' of undefined` upon changing to the page with the isotope instances and they no longer function when trying to add/remove elements. I don't understand why, but after getting this error I can longer change routes with v-to either. I seems to be related to having multiple vue-isotope instances.